### DISCUSSION TITLE
Welcome to Katzebase Discussions!

### DISCUSSION THREAD

**Discussion by NTDLS** [NTDLS/Katzebase#53] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ingted** [comment]
@ingted  => This is Anibal Yeh, using .net for Data Engineering jobs. Recently we want to do some real-time DW / LLM RAG stuffs, and Katzebase seems one perfectly requirements-aligned product. So I would Iike to introduce it into our team. I am only a RDBMS user, not RDBMS theory expert, so I would try to learn more here and try to help to develop as enough as possible test cases to make the development easier, more creative and maybe... ultra fast? haha

BTW, for some part which not too academic, I hope I could join to contribute some code as well...

**Comment by NTDLS** [comment]
My name is Josh Patterson, I've been programing in C/C++ since the early 90s and then C# from 2002. My expertise (or maybe just my passion) has always been building frameworks, engines and low-level processes (like network load-balancers, routers, thread-pools, etc.). For instance, I do not like making games - but I can happily burn weeks of sleepless nights building game engines. Maybe I'm not the best at it, but it gives me that dopamine kick which I need. 😄 

Katzebase is the 4th rewrite, started Apr 24, 2018, of a proprietary RDBMS, first written in C++ (called [NTDBS](https://github.com/NTDLS/NTDBS) (2002), then LeafSQL, then Dokdex, and finally Katzebase). The name change to Katzebase was intended to be more fun and convey the real purpose of the project, which is to enjoy the process of developing a fully fleged ACID compliant database engine and its accompanying tools.

I spend my working hours (1) at a full-time IT management position, (2) running a database performance tuning company ([Vroom Performance Technologies](https://vroomperformance.com/)), and (3) doing open-source development.
